src/BlazorUI/Bit.BlazorUI/Components/Layouts/Stack/BitStack.scss

```
@@ -1,5 +1,11 @@
 ﻿@import "../../../Styles/functions.scss";
+@import "../../../Styles/media-queries.scss";
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)